src/mono/System.Private.CoreLib/src/System/RuntimeMethodHandle.cs

```diff
@@ -90,7 +90,25 @@ internal bool IsNullHandle()
             return value == IntPtr.Zero;
         }
 
-        // Temporary placeholder until Mono adds support for supporting boxing true Nullables.
-        internal static object? ReboxFromNullable(object? src) => src;
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        internal static extern void ReboxFromNullable (object? src, ObjectHandleOnStack res);
+
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        internal static extern void ReboxToNullable (object? src, QCallTypeHandle destNullableType, ObjectHandleOnStack res);
+
+        internal static object ReboxFromNullable (object? src)
```

Comment:
Why are these methods in RuntimeMethodHandle ?

Reply:
Just because they live in the same class on CoreCLR side